core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/html/dot-seo-meta-tags.service.ts

```diff
@@ -621,7 +642,19 @@ export class DotSeoMetaTagsService {
 
         if (
             twitterDescription &&
-            twitterDescription.length < SEO_LIMITS.MAX_TWITTER_DESCRIPTION_LENGTH
+            twitterDescription.length < SEO_LIMITS.MIN_TWITTER_DESCRIPTION_LENGTH
+        ) {
+            result.push(
+                this.getWarningItem(
+                    this.dotMessageService.get('seo.rules.twitter-card-description.less')
+                )
+            );
+        }
+
+        if (
```

Comment:
is this condition correct ? both times asking for `MAX_TWITTER_DESCRIPTION_LENGTH ` ?